webapp/app/router.js

```diff
@@ -6,6 +6,8 @@ const Router = EmberRouter.extend({
   rootURL: config.rootURL
 });
 
+
+/* eslint-disable max-nested-callbacks */
```

Comment:
Oops I made a mistake on merging the conflicts. After this is fixed, the PR will be good to merge 👍